Lib/test/test_argparse.py

```diff
@@ -7128,7 +7128,9 @@ class TestColorized(TestCase):
     def setUp(self):
         super().setUp()
         # Ensure color even if ran with NO_COLOR=1
+        original_can_colorize = _colorize.can_colorize
         _colorize.can_colorize = lambda *args, **kwargs: True
+        self.addCleanup(setattr, _colorize, 'can_colorize', original_can_colorize)
```

Comment:
This can perhaps be written as (not tested):

```suggestion
        self.enterContext(support.swap_attr(_colorize, 'can_colorize',
                                            lambda *args, **kwargs: True))
```
(or using the `unittest.mock` module if you prefer).